tests/functional/dbt_runner/test_dbt_runner.py

```diff
@@ -12,20 +13,20 @@ def dbt(self) -> dbtRunner:
         return dbtRunner()
 
     def test_group_invalid_option(self, dbt: dbtRunner) -> None:
```

Comment:
Minor style/readability suggestion. You could use `pytest.mark.parameterize` here to group these test cases, basically saying "all of these scenarios result in `DbtUsageException`:
```python
@pytest.mark.parameterize("invoke_args",[
    (["--invalid-option"]),
    (["deps", "--invalid-option"]),
    (["--warn-error", "--warn-error-options", '{"include": "all"}', "deps"]),
    (["--invalid-command"]),
])
def test_commands_raise_usage_exception(self, dbt: dbtRunner, invoke_args: list) -> None:
    res = dbt.invoke(invoke_args)
    assert type(res.exception) == DbtUsageException
```